pandas/tests/series/test_operators.py

```diff
@@ -741,6 +741,20 @@ def test_op_duplicate_index(self):
         expected = pd.Series([11, 12, np.nan], index=[1, 1, 2])
         assert_series_equal(result, expected)
 
+    def test_divmod(self):
+        a = Series([1, 1, 1, np.nan], index=['a', 'b', 'c', 'd'])
```

Comment:
Add the GitHub issue number as a comment.

Reply:
Will do.